jenkins.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+# make sure the build env is up to date
+make -C docker
+
+TAG=`cat docker/tag`
+
+if [ -z `docker images -q $TAG` ]; then
+  # the image is not present, so pull or build
+  docker pull $TAG || make -C docker rebuild
+fi
+
+./build.sh
+[ "$1" == "--no-sign" ] || ./sign.sh
```

Comment:
Could we move this whole script to commands in Jenkinsfile?

Reply:
if this is running right now, I'd be ok with leaving this as a follow-on (but def good to do)